### PR TITLE
add Litlyx analytics

### PR DESCRIPTION
**Pull request by botkalista**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/litlyx-analytics.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+name: "Litlyx"
+website_url: "https://litlyx.com"
+source_code_url: "https://github.com/Litlyx/litlyx"
+description: "All-in-one Analytics Solution. Setup in 30 seconds. Display all your data on an AI-powered dashboard. Fully self-hostable and GDPR compliant."
+licenses:
+  - Apache-2.0
+platforms:
+  - Nodejs
+  - Javascript
```
Comment (kokomo123) [inline software/litlyx-analytics.yml:10]:
```suggestion
```
As long as you have Nodejs, it's fine to remove this line 💯

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here, pending other thoughts from collaborators. Thanks for your contribution!

File: software/litlyx-analytics.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+
```
Comment (Rabenherz112) [inline software/litlyx-analytics.yml:1]:
```suggestion
```

File: software/litlyx-analytics.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+name: "Litlyx"
+website_url: "https://litlyx.com"
+source_code_url: "https://github.com/Litlyx/litlyx"
+description: "All-in-one Analytics Solution. Setup in 30 seconds. Display all your data on an AI-powered dashboard. Fully self-hostable and GDPR compliant."
+licenses:
+  - Apache-2.0
+platforms:
+  - Nodejs
```
Comment (Rabenherz112) [inline software/litlyx-analytics.yml:9]:
```suggestion
```
Self-hosting docs only show docker as a possible option to self-host this tool
https://docs.litlyx.com/self-hosting/introduction

Comment (kokomo123) [reply software/litlyx-analytics.yml:9]: image
you can, its in the repo description

**Review by Rabenherz112 — APPROVED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**